### ISSUE TITLE
nchash dependency

### ISSUE THREAD

**Issue by marshallward** [aidanheerdegen/yamanifest#11]
This is more of a discussion point, wanted to get some feedback.

yamanifest is a very lightweight module, except for this one dependency on nchash.  Because of nchash, we need both python-netCDF4 and numpy, which are both very much not lightweight modules.

In particular, numpy installs are big and slow, and can make it hard to keep things lightweight.

Mostly I wanted to see how much nchash is used.  But also, what do you think of making it an optional hash rather than mandatory?

**Comment by aidanheerdegen** [comment]
Absolutely. Don't really use it anymore, but not sure how much work it would be to make it optional. Happy to take PRs!

:)
